### ISSUE TITLE
Organisation users can't add entries

### ISSUE THREAD

**Issue by marekbell** [dani-garcia/bitwarden_rs#261]
I created an organization and invited another user.  Despite given the user management access to several collections the other user can not add any items.  Whenever they attempt to add an item they just get an error popup that simply states an error occurred and gives no further detail.

**Comment by moqmar** [comment]
I have no idea if this is related, but I get a collection named `[error: cannot decrypt]` in my newly created organization, and if I try to save a newly created item, nothing happens (the popup just stays open).

I also cannot create a new collection in my organization, and deleting `[error: cannot decrypt]` doesn't change anything at all.

**Comment by mprasil** [comment]
@marekbell, are you using latest image? Also did you confirm the user after inviting them?

@moqmar, are you using the `SERVER_ADMIN_EMAIL` by any chance?

**Comment by marekbell** [comment]
I just duplicated the same issue on a fresh install.  I'm using the latest Docker image... the website shows the version as '2.5.0'.

To reproduce it on the fresh install I did...

1. Create new user a@mydomain.com.
2. User A creates organization Test and adds an item to the default category within the organization.
3. User A creates second category TestCat in organization and adds item in that.
4. User A invites b@mydomain.com to the organization as Manager with access to categories Default and TestCat.
5. User B joins.
6. User A accepts B.
7. User B can now see the organization and the items in both the Default and TestCat categories.
8. User B attempts to add a new item to either Default or TestCat.  Error shown states "An error has occurred.  An unexpected error has occurred".

So, user B can't add anything to the organization.  User B is able to make their own items without error, but can not make items that belong to the organization.

**Comment by marekbell** [comment]
Just tried again and when I look in the log immediately after trying step 8 above I see...

POST /api/ciphers/create application/json; charset=utf-8: | stdout
=> Matched: POST /api/ciphers/<uuid> | stdout
=> Error: Couldn't parse JSON body: Error("missing field `Type`", line: 1, column: 607) | stdout
=> Outcome: Failure | stdout
=> Warning: Responding with 400 Bad Request catcher. | stdout

**Comment by moqmar** [comment]
@mprasil You're right, thank you. Seems like I didn't fully read the README, sorry for that! :see_no_evil:
